command/meta_backend.go

```diff
@@ -390,16 +390,14 @@ func (m *Meta) backendFromConfig(opts *BackendOpts) (backend.Backend, error) {
 	case c != nil && s.Remote.Empty() && !s.Backend.Empty():
 		// If our configuration is the same, then we're just initializing
 		// a previously configured remote backend.
-		if !s.Backend.Empty() {
```

Comment:
Why are we changing the logic here?
I can't recall exactly why I put that in, and the test for it still passes, but the logic around the backend state is quite finicky and I'd rather not change any of it at if at all possible.

Reply:
oh never-mind, I didn't put that in, and I see how it's redundant wit the check above ;)

Reply:
Ha sorry, I realized after I pushed that should've been in its own commit with a comment!